### DISCUSSION TITLE
Chroma getOrCreateCollection error:

### DISCUSSION THREAD

**Discussion by omar-ayoub** [FlowiseAI/Flowise#625] [Q&A]
i get this error:
Error: Chroma getOrCreateCollection error: Error: TypeError: fetch failed

i'm using local "local ai embedding", "chat local ai" with csv file , the local embedding and local ai chat is working i can see processing monitor from docker that the image of the llm is processing but the not the image of chroma db this error shows up from chroma db in the chat box of flowise, any idea why this happens, (text splitiing is 1000,200 for chunk size and chunk overlap)
iam using flowise locally not in docker

**Comment by chungyau97** [comment]
Hi @omar-ayoub,
I believe it's related to dimensions, 

Troubleshoot steps:
1. LocalAI is `384`, you can try this using [Pinecone](https://docs.flowiseai.com/vector-stores/pinecone).
2. Swap LocalAI to OpenAI for chat model and embedding.
